src/index.html

```diff
@@ -6,13 +6,61 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product card</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/main.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="/src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        class="card__picture"
+      />
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__rating">
+        <div class="card__rating-stars">
+          <div class="card__rating-star"></div>
+          <div class="card__rating-star"></div>
+          <div class="card__rating-star"></div>
+          <div class="card__rating-star"></div>
+          <div class="card__rating-star"></div>
+        </div>
```

Comment:
<img width="486" alt="image" src="https://github.com/user-attachments/assets/4438f577-2769-452e-869a-b755c7b88871">

I should have `stars--n` modifier, the same as in previous task